projects/aca-folder-rules/src/lib/manage-rules/manage-rules.smart-component.html

```diff
@@ -0,0 +1,56 @@
+<aca-page-layout>
+
+  <aca-page-layout-header>
+    <adf-toolbar class="adf-toolbar--inline">
+      <button mat-icon-button (click)="goBack()">
+        <mat-icon>arrow_back</mat-icon>
+      </button>
+    </adf-toolbar>
+    <adf-breadcrumb root="{{'ACA_FOLDER_RULES.ACTIONS.MANAGE_RULES' | translate}}"></adf-breadcrumb>
+  </aca-page-layout-header>
+
+  <aca-page-layout-content>
+    <div class="main-content">
+
+      <ng-container *ngIf="isLoading$ | async; else onLoaded">
+        <mat-progress-bar color="primary" mode="indeterminate"></mat-progress-bar>
+      </ng-container>
+
+      <ng-template #onLoaded>
+        <ng-container *ngIf="folderInfo; else genericError">
+          <adf-toolbar class="adf-toolbar--inline aca-rules-actions-bar">
+            <adf-toolbar-title>
+              <mat-icon>folder</mat-icon>
+              <adf-breadcrumb root="{{folderInfo.name}}:{{'ACA_FOLDER_RULES.TOOLBAR.BREADCRUMB.RULES' | translate}}"
+                              class="aca-rules-breadcrumb"></adf-breadcrumb>
+            </adf-toolbar-title>
+          </adf-toolbar>
+          <mat-divider></mat-divider>
+
+          <div class="aca-manage-rules-container" *ngIf="(rules$ | async).length > 0 ; else emptyContent">
+            <aca-rules-list [rules]="rules$ | async"></aca-rules-list>
+            <aca-rule-details [readOnly]="true"></aca-rule-details>
```

Comment:
There is no value filled in here. However I realise now that my rule details component currently only takes an `initialValue`, rather than an input property you can actually change, so that's my bad. Maybe provide an `[initialValue]` here anyway with the value of the selected rule, and I'll change it so it actually changes the value of the form.